visualization_scripts/style.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<vizmap id="VizMap-2019_11_22-16_53" documentVersion="3.0">
+    <visualStyle name="default">
```

Comment:
Can we name this `TPS` or something more descriptive?

Reply:
This is the only change I recommend in the style file.  Everything else looks good.  You should be able to manually edit the file.